### PR TITLE
Ikuffour

### PR DESCRIPTION
**Pull request by ikuffourosei**
- Deleted the empty Frontend Directory.
- Since our model is based on MVC,I moved all the VIEW components of the web app into `view` directory since that is how `Jinja2` from `Flask` renders HTML pages. Same with `Django`, they all look for HTML files in a folder called templates, none other. Tried different methods to render outside templates, didn't work
- Added some HTML files and CSS styles to render the home page with flask.
- Home page screenshot below
- 
rentopia_page

### CODE REVIEW

**Comment by ikuffourosei** [comment]
Searched on stack overflow and chatgpt if there was a way to render HTML files from a different directory outside templates, I tried them but they al don't work. Unless I change the name to templates and it must be in the same folder as the flask app

File: view/__pycache__/app.cpython-38.pyc
Comment (angelotheman) [inline view/__pycache__/app.cpython-38.pyc:1]: Try removing these files before committing them

**Review by angelotheman — APPROVED** [review]
Checked out the files. I have a bit of issues with the naming conventions used. I'll verify from the flask documentation

**Comment by ikuffourosei** [comment]
Alright. But everything in the view is about routing the frontend or what
the user can see but yea you can check their docs out. or even stack
overflow

**Pull request merged**